src/config/2020.json

```diff
@@ -2,7 +2,7 @@
   "settings": [
     {
       "is_live": false,
-      "supported_languages": ["en","es","fr","ja"],
+      "supported_languages": ["en","es","fr","ja", "ko"],
```

Comment:
It's OK to leave in here for now as 2020 is not live yet. But we'll probably remove it later as probably will launch English first.